db/migrate/20221221073259_sanitize_user_accessibility_in_users.rb

```diff
@@ -0,0 +1,21 @@
+class SanitizeUserAccessibilityInUsers < ActiveRecord::Migration[6.1]
+  def up
+    User.all.each do |user|
```

Comment:
@Pralish @alis-khadka what's happening here? Looks like the granular permission scheme for API is being changed to accommodate this feature

Reply:
Before introducing `ApiKey`, all other API entities would fall under an `ApiNamespace`, i.e.: with all other API entities, ApiNamespace had `has many` relation. Due to this, the value of column `api_accessibility` was tailored considering `ApiNamespace`.
```
{ all_namespaces: {full_access: 'true'}}
OR
{namespaces_by_category: {category_name: {full_access: 'true'}}}
```

With the introduction of `ApiKey`, it has `many-to-many` relation with `ApiNamespace` due to which access related to `ApiKey` would not fall under a single `ApiNamespace`. Since,  the value of column `api_accessibility` was tailored considering `ApiNamespace` only, it would be better to organize `api_accessibility` under 2 categories of access; 1) api_namespace  2) api_key. 
```
{
api_namespaces: { all_namespaces: {full_access: 'true'}},
api_keys: {full_access: 'true'}
}
```

This migration file is basically moving the current value of `api_accessibility` under  `api_namespace` key.